package.json

```diff
@@ -115,18 +115,19 @@
 		"minami": "^1.2.3",
 		"mkdirp": "^0.5.1",
 		"mocha": "^6.1.2",
-		"prettier": "^1.17.0",
+		"prettier": "^1.14.0",
```

Comment:
why did this go backward 3 versions?

Reply:
Because I had to not accept the prettier changes since it conflicted all my test/integration html files, but i hadn't realized it also included a `package.json` change that actually made prettier run on the html files. So even though I reran `fmt`, it didn't do anything. Fixed in my latest commit.

Reply:
We need Prettier v15 or better to support HTML.